### PR TITLE
Draft of Release Notes

### PR DESCRIPTION
**Pull request by cdsupina**
I made a new chapter in the book called "Releases" where we can track the history of changes made to the game. I made a new section in this chapter called "0.2" (I wasn't sure what the release number would be) that contains the release notes. These can be copied into the Github release as well.

Let me know if there is anything missing from the release notes that should be included.

I also updated the items section to only include items that are currently active in the game.

### CODE REVIEW

**Review by olefasting — COMMENTED** [review]
Looks good but this will actually be v0.3

v0.2 was never tagged on github, just bumped in our Cargo.toml file. It was the previous release where stuff was reset and the now removed network support was added

**Comment by erlend-sh** [comment]
> You might also want to add the redesigned main menu and the addition of an in-game menu, but that is up to you :)

yeh let’s get those in before merging.

File: book/src/09-12-2021.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  - Cannon
+  - Crate
+  - Grenades
+  - Kick-Bomb
+  - Machine Gun
+  - Mines
+  - Sniper Rifle
+  - Sword
+  - Turtle Shell
+- New Characters!
```
Comment (tigleym) [inline book/src/09-12-2021.md:19]: Perhaps we can include the sprites for these characters?

**Review by tigleym — APPROVED** [review]
This looks good to me! Thanks.

**Pull request merged**